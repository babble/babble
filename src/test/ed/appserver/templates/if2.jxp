<if ( 5 == 2 )>
<div>
  a
</div>
</if>

<if ( 5 == 5 )>
<div>
  b
</div>
</if>

