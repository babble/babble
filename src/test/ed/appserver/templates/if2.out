<div>
  b
</div> LINE26

<div>
  <span>
    this is fun
  </span> LINE33
</div>
